lib/models/user.js

```diff
@@ -94,6 +94,7 @@ User.prototype.githubLogin = function (accessToken, cb) {
           }
           self.id('me');
           self.fetch(cb);
+          self.connectSid = res.headers['set-cookie'][0].split(';')[0].split('=')[1];
```

Comment:
Also, if this fails it's the last thing in the list so if the stack dies it's totally fine